### PR TITLE
Data validation

### PR DESCRIPTION
**Pull request by zelenins**
Added data validation to account-service, checking data on create and update

### CODE REVIEW

File: functions/account.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+createAccount:
+  handler: src/handler/account-handler.createAccount
+  events:
+    - http:
+        path: accounts/create
```
Comment (jpanticlevi9) [inline functions/account.yml:5]: path should be just "accounts", without "/create"

File: src/service/account-service.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    ExpressionAttributeValues: {
+      ":name": accountData.name,
+      ":email": accountData.email,
+      ":status": accountData.status,
+      ":IAMUsers": accountData.IAMUsers
+    },
+    ReturnValues: "ALL_NEW"
+  }
+  const updatedAccount = dynamoDB.update(params).promise();
+  return updatedAccount;
```
Comment (jpanticlevi9) [inline src/service/account-service.js]: you don't need this constant, you can just return dynamoDB.update(params).promise();

File: functions/account.yml (hunk 1 above)
Comment (zelenins) [reply functions/account.yml:5]: Changed in a previous PR by Marko.

File: src/service/account-service.js (hunk 2 above)
Comment (zelenins) [reply src/service/account-service.js]: Done

**Review by jpanticlevi9 — APPROVED** [review]

**Review by markoninho — APPROVED** [review]

**Pull request closed without merging**